### DISCUSSION TITLE
How to implement htpasswd?

### DISCUSSION THREAD

**Discussion by cod3rshotout** [evertramos/nginx-proxy-automation#393] [Q&A]
Hello, I followed [this instructions](https://github.com/tlvince/docker-compose-letsencrypt-nginx-proxy-companion/blob/693a7acd9f1a293d5847e7419060ef43c54030ad/.github/README.md#further-options) so I was able to create the file ".htpasswd" within the following folder: "proxy/data/htpasswd/example.com/.htpasswd".

I exec in the proxy-web-auto container and launched the following command "nginx -s reload", but my site isn't protected yet.

Could someone help me on this? Thanks.

**Comment by evertramos** [comment]
Hello @cod3rshotout,

Instead of `proxy/data/htpasswd/example.com/.htpasswd`, try:

`proxy/data/htpasswd/example.com`

Where the example.com will be a file named over your domain and will have the user and password.

**Comment by cod3rshotout** [comment]
> Hello @cod3rshotout,
> 
> Instead of `proxy/data/htpasswd/example.com/.htpasswd`, try:
> 
> `proxy/data/htpasswd/example.com`
> 
> Where the example.com will be a file named over your domain and will have the user and password.

Thanks for the reply, I tried as you suggested then I exec within the nginx container using "docker exec -it proxy-web-auto" and launched: "nginx -s reload". Nothing has changed. Am I missing something?
